Makefile

```diff
@@ -22,10 +22,7 @@ release:
 
 .PHONY: $(TARGETS)
 
-.PHONY: generate
-generate: build/data 
-	./scripts/download
-	go generate
+generate: download
```

Comment:
Why take off the `.PHONY`?

Reply:
Because generate is a script target now and is already phony. The Makefile is in pretty bad shape and needs to be re-done anyways.

Reply:
Cool. Thanks!